lib/beaker/options/presets.rb

```diff
@@ -13,7 +13,7 @@ class Presets
       # so that preferred and "fallback" values work as expected.
       ENVIRONMENT_SPEC = {
         :home                 => 'HOME',
-        :project              => ['BEAKER_PROJECT', 'BEAKER_project'],
+        :project              => ['BEAKER_PROJECT', 'BEAKER_project', 'JOB_NAME'],
```

Comment:
Like BUILD_URL a few lines below, this comes from Jenkins and is used to set the project name if BEAKER_PROJECT (or _project) is unset.